package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@waxio/waxjs",
-  "version": "1.5.0",
+  "version": "1.5.1-beta",
```

Comment:
should be 1.6.0-beta since this adds a new feature. Incrementing the last in the triple is done for bug fixes only, the second for backwards comaptible new features, and the first one for backwards incompatible upgrades